### DISCUSSION TITLE
laudala database idi!

### DISCUSSION THREAD

**Discussion by sriharshaguthikonda** [milvus-io/milvus#34856] [Q&A and General discussion]
doing anything with this is almost impossible...!.....

**Comment by xiaofan-luan** [comment]
Hi @sriharshaguthikonda 
could you give more detailed feedback?
what is the problem you are facing? What is the scale and use case? what is the error you are facing and is there any log?

**Reply by sriharshaguthikonda** [reply]
figured it out.! tq
